example/satosa/integration_test/settings.py

```diff
@@ -20,7 +20,8 @@
             "module": "pyeudiw.storage.mongo_storage",
             "class": "MongoStorage",
             "init_params": {
-                "url": "mongodb://localhost:27017/",
+                # according to Satosa-Saml2Spid demo
+                "url": "mongodb://satosa:thatpassword@localhost:27017/?timeoutMS=2000",
```

Comment:
```suggestion
                "url": f"mongodb://{os.getenv('PYEUDIW_MONGO_TEST_AUTH_INLINE', '')}localhost:27017/?timeoutMS=2000",
```
example: `PYEUDIW_MONGO_TEST_AUTH_INLINE="satosa:thatpassword@"`